frontend/packages/kubevirt-plugin/src/components/virtualization-overview/status-card/constants.ts

```diff
@@ -0,0 +1,14 @@
+export const ODF_NAME = 'OpenShift Data Foundation (ODF)';
+export const LSO_NAME = 'Local storage (LSO)';
+
+export enum ClusterServiceVersionPhase {
```

Comment:
I took this from the OLM package and it had all of them. I figured we can leave them there in case we decide to account for other statuses at some point. It's also possible that we'll need this for other features, in which case we can just move it as is to a common location and it will already have all of the phases.